OpenSimRT/RealTime/src/experimental/RealTimeAnalysisExtended.cpp

```diff
@@ -0,0 +1,189 @@
+/**
+ * -----------------------------------------------------------------------------
+ * Copyright 2019-2021 OpenSimRT developers.
+ *
+ * This file is part of OpenSimRT.
+ *
+ * OpenSimRT is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OpenSimRT is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * OpenSimRT. If not, see <https://www.gnu.org/licenses/>.
+ * -----------------------------------------------------------------------------
+ */
+#include "RealTimeAnalysisExtended.h"
```

Comment:
Maybe think of a better name for this pipeline. Also, it might be better to give another name to RealTimeAnalysis as well. Can you propose something?

Reply:
Not really.. It's based on the Facade design pattern, so I'd only suggest a name that includes this, so it is more obvious to the user. Like, 'PipelineAnalysisFacade' or similar..